### PR TITLE
migrate code example and doc

### PR DESCRIPTION
**Pull request by baiyfbupt**
### PR types

Others
### PR changes

APIs
### Describe

1. Migrate `adaptive_pool2d`|`adaptive_pool3d`|`mse_loss`|`square_error_cost` code example and doc to API 2.0
2. Remove `adaptive_pool2d`|`adaptive_pool3d` API alias, because `adaptive_(avg/max)_(pool2d/pool3d)` APIs have been added in API 2.0 already.

### CODE REVIEW

**Comment by baiyfbupt** [comment]
Will fix `core.ops` issue in next PR.

**Comment by zhiqiu** [comment]
> Will fix `core.ops` issue in next PR.

LGTM

**Review by zhiqiu — DISMISSED** [review]

File: python/paddle/fluid/layers/loss.py [hunk 1]
```diff
@@ -316,49 +316,26 @@ def square_error_cost(input, label):
...
-    		input = dg.to_variable(input_data)
-    		label = dg.to_variable(label_data)
-    		output = fluid.layers.square_error_cost(input, label)
-    		print(output.numpy())
-	        
-	        # [0.04000002]
+            import paddle
+            input = paddle.to_tensor([1.1, 1.9])
+            label = paddle.to_tensor([1.0, 2.0])
+            output = paddle.fluid.layers.square_error_cost(input, label)
```
Comment (TCChenlong) [inline python/paddle/fluid/layers/loss.py]: 这里用推荐别名哈，应该是 paddle.nn.functional.square_error_cost 这个吧？麻烦确认下

File: python/paddle/fluid/layers/loss.py [hunk 2]
```diff
@@ -316,49 +316,26 @@ def square_error_cost(input, label):
         Out = (input - label)^2
 
     Parameters:
```
Comment (TCChenlong) [inline python/paddle/fluid/layers/loss.py:315]: 305-3.7 alias相关的内容可以删除

File: python/paddle/fluid/layers/loss.py [hunk 3]
```diff
@@ -1790,47 +1764,23 @@ def mse_loss(input, label):
...
-    		label = dg.to_variable(label_data)
-    		output = fluid.layers.mse_loss(input, label)
-    		print(output.numpy())
-	        
-	        # [0.04000002]
 
+            import paddle
+            input = paddle.to_tensor([1.1, 1.9])
+            label = paddle.to_tensor([1.0, 2.0])
+            output = paddle.fluid.layers.mse_loss(input, label)
```
Comment (TCChenlong) [inline python/paddle/fluid/layers/loss.py:1778]: 同上 应该是 paddle.nn.functional.mse_loss 吧？

File: python/paddle/fluid/layers/loss.py (hunk 1 above)
Comment (baiyfbupt) [reply python/paddle/fluid/layers/loss.py]: done

File: python/paddle/fluid/layers/loss.py (hunk 2 above)
Comment (baiyfbupt) [reply python/paddle/fluid/layers/loss.py:315]: removed

File: python/paddle/fluid/layers/loss.py (hunk 3 above)
Comment (baiyfbupt) [reply python/paddle/fluid/layers/loss.py:1778]: paddle.nn.functional.mse_loss 是一个新的API，与这里不同，是这个API的升级版。
这里示例代码应该用老的api

**Review by zhiqiu — APPROVED** [review]

**Review by TCChenlong — APPROVED** [review]
LGTM
